constraint/pkg/client/e2e_test.go

```diff
@@ -866,21 +946,69 @@ func TestE2E_Review_RegoEvaluationMeta(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	results := responses.Results()
+	for _, se := range responses.StatsEntries {
+		if se.Scope == instrumentation.TemplateScope {
+			if se.Stats == nil {
+				t.Errorf("missing stats for: %v missing stats", se.StatsFor)
+			}
 
-	// for each result check that we have the constraintCount == 3 and a positive templateRunTime
-	for _, result := range results {
-		stats, ok := result.EvaluationMeta.(rego.EvaluationMeta)
-		if !ok {
-			t.Fatalf("could not type convert to RegoEvaluationMeta")
-		}
+			for _, stat := range se.Stats {
+				// now check that descriptions are able to be fetched.
+				desc := c.GetDescriptionForStat(instrumentation.RegoSource, stat.Name)
+
+				switch stat.Name {
+				case "templateRunTimeNanos":
```

Comment:
Testing stat -> description should be a different test.

Reply:
globbed into the `cmp` usage